simple_make.sh

```diff
@@ -13,15 +13,19 @@ elif which pacman; then
         git base-devel qt5 openal libxss qrencode opus libvpx libsodium
 elif which dnf; then
     sudo dnf group install \
-        "Development Tools"
+        "Development Tools" \
+        "C Development Tools and Libraries"
     # pure Fedora doesn't have what it takes to compile qTox (ffmpeg)
     sudo dnf install \
         http://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm
     sudo dnf install \
         git qt-devel qt-doc qt-creator qt5-qtsvg qt5-qtsvg-devel \
         openal-soft-devel qt5-qttools-devel libXScrnSaver-devel \
         qrencode-devel opus-devel libvpx-devel glib2-devel gdk-pixbuf2-devel \
-        gtk2-devel libsodium-devel ffmpeg-devel sqlite sqlite-devel
+        gtk2-devel libsodium-devel ffmpeg-devel sqlite sqlite-devel libtool \
+        openssl-devel
+    # Fedora by default doesn't include libs in /usr/local/lib so add it
+    echo '/usr/local/lib/' | sudo tee -a /etc/ld.so.conf.d/locallib.conf
```

Comment:
`is_locallib()` checks whether line `/usr/local/lib/` exists in supplied file.

Only if line doesn't exist (non-zero exit status of `grep`) it is being appended to the file.

I didn't really write it well – confusing function & variable names, etc, just wanted to express the idea in which direction it should go.

`/usr/local/lib/` also should be a variable, for that though I'd need to test whether I'm escaping correctly `grep` regexp, so I just left it as is.